client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -319,15 +318,9 @@ const PluginsBrowser = React.createClass( {
 		);
 
 		if (
-			( this.state.accessError || cantManage ) &&
-			(
-				// If automated transfer is _off_ then behave
-				// as normal. If it's on, then only show if we
-				// are getting an error on a Jetpack site
-				! this.props.atEnabled ||
-				( selectedSite && selectedSite.jetpack )
-			)
-		) {
+			( this.state.accessError || cantManage ) && selectedSite
```

Comment:
minor nitpick: now that this is down to a one liner conditional it would be easier to read as: 
    if ( ( this.state.accessError || cantManage ) && selectedSite ) ) {
      ...
    }